blaze/expr/strings.py

```diff
@@ -83,6 +113,86 @@ class str_lower(UnaryStringFunction):
     def schema(self):
         return self._child.schema
 
+class str_isalnum(UnaryStringFunction): schema = bool_
+class str_isalpha(UnaryStringFunction): schema = bool_
+class str_isdecimal(UnaryStringFunction): schema = bool_
+class str_isdigit(UnaryStringFunction): schema = bool_
+class str_islower(UnaryStringFunction): schema = bool_
+class str_isnumeric(UnaryStringFunction): schema = bool_
+class str_isspace(UnaryStringFunction): schema = bool_
+class str_istitle(UnaryStringFunction): schema = bool_
+class str_isupper(UnaryStringFunction): schema = bool_
+
+class StrFind(ElemWise):
+    """
+    Find literal substring in string column.
+
+    """
+
+    __slots__ = '_hash', '_child', 'sub'
```

Comment:
In a recent change this has changed to `_arguments` and should not include the `_hash`. These are just the names of the arguments to the node. More info is here: https://github.com/blaze/blaze/blob/master/docs/source/expr-compute-dev.rst#adding-expressions